code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -874,24 +875,43 @@
 	Read_Memory()
 	if(rounds_survived == longest_survival)
 		speak += pick("...[longest_survival].", "The things I've seen!", "I have lived many lives!", "What are you before me?")
+		desc += " Old as sin, and just as loud. Claimed to be [rounds_survived]."
 		speak_chance = 20 //His hubris has made him more annoying/easier to justify killing
+		color = "#EEEE22"
+	else if(rounds_survived == longest_deathstreak)
+		speak += pick("What are you waiting for!", "Violence breeds violence!", "Blood! Blood!", "Strike me down if you dare!")
+		desc += " The squawks of [rounds_survived * -1] dead parrots ring out in your ears..."

```

Comment:
I wasn't 100% sure that would work, even though it should work.

Because you know, byond.